### PR TITLE
Portfolio: Mobile Version

### PR DESCRIPTION
**Pull request by DhuruvSingh**
## Mobile Version

1. **Added Work Section**
2. **Added About My Self Section**
3. **Added All Necessary Components specified in README file of project**

> **Kindly Review And Give Your Valuable Feedback**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,8 +60,242 @@ <h1>Hey there. I’m Dhuruv <br><span class="h-green">I’m a software developer
...
+                <h1>About me</h1>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+                <button type="button">Get My Resume</button>
+            </div>
+            <div class="about-me-illustration">
+                <img class="img-1" src="./icons/about-me-illustration.png" alt="about me illustraion">
+                <img class="img-2" src="./icons/about-me-illustration2.png" alt="about me illustraion">
```
Comment (MeqdamAlqudah) [inline index.html:221]: - [ ] Please use the same margins and padding as in Figma desing

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @DhuruvSingh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -60,8 +60,242 @@ <h1>Hey there. I’m Dhuruv <br><span class="h-green">I’m a software developer
...
+        <div class="work-container">
+            <div class="card-item">
+                <div class="text-box">
+                    <h4 class="project-title">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="project-tag">
+                        <li>
+                            <p>Ruby on rails</p>
+                        </li>
+                        <li>
+                            <p>css</p>
```
[the remark this answers is not in the document]
Comment (DhuruvSingh) [reply index.html:85]: Done

File: index.html [hunk 3]
```diff
@@ -60,8 +60,242 @@ <h1>Hey there. I’m Dhuruv <br><span class="h-green">I’m a software developer
...
+                <div class="languages">
+                    <div class="icon">
+                        <img src="./icons/icon-languages.png" alt="languages icon">
+                    </div>
+                    <div class="about-title-tag">
+                        <h4>Languages</h4>
+                        <ul>
+                            <li>
+                                <p>JavaScript</p>
+                            </li>
```
[the remark this answers is not in the document]
Comment (DhuruvSingh) [reply index.html:233]: Done

File: index.html [hunk 4]
```diff
@@ -60,8 +60,242 @@ <h1>Hey there. I’m Dhuruv <br><span class="h-green">I’m a software developer
...
+                    <div class="project-btn-sec">
+                        <button type="button">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="about-me">
+            <div class="about-me-text">
+                <h1>About me</h1>
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
```
[the remark this answers is not in the document]
Comment (DhuruvSingh) [reply index.html:213]: done

File: index.html (hunk 1 above)
Comment (DhuruvSingh) [reply index.html:221]: I checked everything looks fine margin and padding

**Review by sumairq — APPROVED** [review]
### Hi @DhuruvSingh  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**